visual-js/visual-snapshots/src/app/worker/worker.ts

```diff
@@ -16,7 +25,16 @@ import { VisualSnapshotsApi } from "../../api/visual-snapshots-api.js";
 program
   .addOption(usernameOption)
   .addOption(accessKeyOption)
-  .addOption(regionOption);
+  .addOption(regionOption)
+  .addOption(buildNameOption)
+  .addOption(branchOption)
+  .addOption(defaultBranchOption)
+  .addOption(projectOption)
+  .addOption(buildIdOption)
+  .addOption(customIdOption)
+  .addOption(suiteNameOption)
+  .addOption(testNameOption)
+  .addOption(snapshotNameOption);
```

Comment:
This shouldn't be necessary here. All snapshot options are passed to the function. Did something not work here?

Reply:
Using any of those arguments resulted in a failure caused by an invalid argument. Please try it yourself. I tried with `--test-name` and `--suite-name` options.

Reply:
Ah! Then maybe a good idea would be to add instead this?

```ts
  .allowUnknownOption(true)
```

Reply:
It works :)